careers/blocks/video/video.css

```
@@ -0,0 +1,44 @@
+.video.block {
+  position: relative;
+  width: 100%;
+  height: 560px;
+  border: 1px solid red;
```

Test border can be removed and hide any potential overflow.

```suggestion
  overflow: hidden;
```